beta/examples/tensorflow/classification/main.py

```diff
@@ -70,8 +70,7 @@ def get_config_from_argv(argv, parser):
     return config
 
 
-def get_dataset_builders(config, strategy, one_hot=True):
-    num_devices = strategy.num_replicas_in_sync if strategy else 1
+def get_dataset_builders(config, num_devices, one_hot=True):
```

Comment:
For what reason are you changing the signature of the get_dataset_builders function?

Reply:
These minor changes are needed to make dependencies more explicit.